pronto_quadruped_ros/include/pronto_quadruped_ros/bias_lock_handler_ros.hpp

```diff
@@ -89,23 +89,20 @@ ImuBiasLockBaseROS<JointStateT>::ImuBiasLockBaseROS(ros::NodeHandle& nh) : nh_(n
 
   nh_.getParam(ins_param_prefix + "frame", imu_frame);
   std::string base_frame = "base";
+  nh.param<std::string>("base_link_name", base_frame, "base");
```

Comment:
better print a `ROS_WARN` in case the parameter is not filled and/or a `ROS_INFO` with what the parameter ends up being